.circleci/config.yml

```
@@ -6,7 +6,7 @@ jobs:
     steps:
       - checkout
       - run: flutter doctor
-      - run: pub get
+      - run: dart pub get
```

The suggestion is solely meant for readability (both achieve the same result), we're already using the `flutter` command with the other run statements.

```suggestion
      - run: flutter pub get
```